### DISCUSSION TITLE
Meeting list relative to user time zone or site time zone?

### DISCUSSION THREAD

**Discussion by willinrecovery** [code4recovery/12-step-meeting-list#1560] [Q&A]
Now that our meetings are largely virtual, our website will be visited by people from many timezones.

https://staging.debtorsanonymous.ca/meetings/?tsml-day=any

Our site is set to Eastern Daylight Savings Time (EDT) and I am in EDT, but it just occurred to me: what happens if someone from Pacific Daylight Savings Time (PDT) visits the site? Is the meeting list relative to the time zone of the visitor or of the site?

For example, we have one meeting at 7:30pm EDT. Will a visitor from Los Angelese see the meeting on the meeting list as 7:30pm or 4:30pm? When I open up the meeting detail, it shows the time zone explicitly, but not in the list.

I just presumed it would convert the time zone local to the site visitor.  Is that the case?

If the list times are in the time zone of the site and not to the site visitor, would it be possible to list the time zone explicitly in the list?

Thank you so much

**Comment by joshreisner** [comment]
hi @willinrecovery your meeting list is set `America/Toronto` at Meetings > Settings, so that's the timezone that's displayed in the meeting list. it is possible to set this to empty (the first value in the list) and it will adjust the listings to the user's timezone, but that's only recommended that for sites that that list meetings across a range of different timezones, such as a national or international list.

<img width="520" alt="Screenshot 2024-11-01 at 2 55 42 PM" src="https://github.com/user-attachments/assets/2035ac3e-fb0d-42db-a297-0966c4e5204a">

it is intuitive for someone coming from another location to see the meetings listed in local time. users in california who come to Philadelphia's AA website, for example, expect the listings to be displayed in ET. 

it is smart about which is the next meeting. for example, a Los Angeles visitor coming to your site at 10am your time tomorrow morning (7am LA time) will see the 10:30am meeting shown as next (just like a local user would). the 9:30am meeting will be "in progress." if they add a meeting to their calendar, it will get added with the right timezone.

we don't think it would be a good use of space to add the timezone in each row of the table. you're welcome to add a notice above the list, but our experience shows this is not necessary. this software is on lots of websites and users are not confused by it
